pkgs/development/libraries/libprom/default.nix

```diff
@@ -0,0 +1,41 @@
+{ gccStdenv
+, fetchFromGitHub
+, cmake
+}:
+let
+  stdenv = gccStdenv;
+in
+stdenv.mkDerivation rec {
+  pname = "libprom";
+  version = "0.1.1";
+
+  src = fetchFromGitHub {
+    owner = "digitalocean";
+    repo = "prometheus-client-c";
+    rev = "v${version}";
+    sha256 = "0g69s24xwrv5974acshrhnp6i8rpby8c6bhz15m3d8kpgjw3cm8f";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  doCheck = false;
+
+  patches = [
+     # Required so CMAKE_INSTALL_PREFIX is honored, otherwise it
+     # installs headers in /usr/include (absolute)
+    ./fix-install-dir.patch
```

Comment:
Is there an upstream issue or pull request? Perhaps suggest [`GNUInstallDirs`](https://cmake.org/cmake/help/v3.0/module/GNUInstallDirs.html) module.

Reply:
@jtojnar https://github.com/digitalocean/prometheus-client-c/pull/20

Reply:
Also could you use `fetchpatch` with `https://github.com/digitalocean/prometheus-client-c/pull/20.patch`?

Reply:
@jtojnar @cole-h That patch does not apply correctly to 0.1.1 I'm afraid.
```
applying patch /nix/store/karlwcmlzva7f9h9y90nqrjah07jg6yl-0c15e7e45ad0c3726593591fdd7d8f2fde845fe3.patch
patching file prom/CMakeLists.txt
Hunk #1 FAILED at 136.
1 out of 1 hunk FAILED -- saving rejects to file prom/CMakeLists.txt.rej
```
Since that patch upstream has been merged already maybe we can just merge this as is and completely remove patching as soon as they release a new version?

Reply:
Could you also `fetchpatch` `https://github.com/digitalocean/prometheus-client-c/commit/5fcedeb506b7d47dd7bab35797f2c3f23db6fe10.patch`? Hopefully, that should make the patch apply and also resolve the `TEST` issue.

Reply:
Done.